jwst/wfs_combine/wfs_combine.py

```diff
@@ -116,7 +135,7 @@ def create_aligned_2(self):
            smoothed image, and taking the mean of the coordinates of these
            pixels.  The x- and y-mean defines the centroid location.
         3. Set the limits of the subarrays for interpolation by taking this
-           centroid +/- PSF_SIZE and adding the BLUR_SIZE, taking the edges
+           centroid +/- psf_size and adding the BLUR_SIZE, taking the edges
```

Comment:
It looks like `BLUR_SIZE` was removed above, so should this commentary be changed?

Reply:
Oh, never mind. `BLUR_SIZE` is no longer a macro defined in the code, but passed in as a parameter.